cbmc_viewer/tracet.py

```diff
@@ -510,7 +517,14 @@ def parse_json_step(step, root=None):
               parse_json_function_return if kind == 'function-return' else
               parse_json_location_only if kind == 'location-only' else None)
     if parser is None:
-        raise UserWarning("Unknown json step type: {}".format(kind))
+        # skip uninteresting kinds of steps
+        if kind in ['loop-head']:
```

Comment:
Are you expecting this list to grow with more kinds of steps?

Reply:
I do expect the list to grow over time.